tests/test_read.py

```diff
@@ -159,13 +159,11 @@ def test_reading_En22k4_instance():  # instance from CVRPLIB
         assert_equal(data.location(loc).required, True)
 
 
-def test_reading_RC208_instance():  # Solomon style instance
+def test_reading_RC208_instance():
```

Comment:
Do we still want to ahve this test now that we only do VRPLIB? I think the test above this already covers VRPLIB; not sure if another test is valuable.

Reply:
Yea I'll remove this one and update the above test a bit.